docs/checkit.js

```diff
@@ -32,7 +32,13 @@ function CheckIt(mode, calendarDiv) {
     this.$endDate = $('#endDate');
     this.$buildCalendarForm = $('#collapseOne');
     this.$loadingWheel = $('#loadingWheel');
-    this.$calendarDiv = calendarDiv;
+    
+    if (calendarDiv.length === 0) {
+        console.error("Selector given for calendar div does not select anything.");
+    }
+    else {
+        this.$calendarDiv = calendarDiv;
```

Comment:
You could treat the case where `calendarDiv` itself is undefined (i.e. the caller didn't pass a second argument) as meaning "I don't want to initialize this yet, i'll do it later. That could be fine.